src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private final int firstLeg;
+    private final double secondLeg;
```

Comment:
It would be better to use the same data type for both legs (either both int or both double). 🧐